doctests/search_vss.py

```diff
@@ -0,0 +1,340 @@
+# EXAMPLE: search_vss
+# HIDE_START
+"""
+Code samples for vector database quickstart pages:
+    https://redis.io/docs/latest/develop/get-started/vector-database/
+"""
+# HIDE_END
+
+# STEP_START imports
+import json
+import time
+
+import numpy as np
+import pandas as pd
+import requests
+import redis
+from redis.commands.search.field import (
+    NumericField,
+    TagField,
+    TextField,
+    VectorField,
+)
+from redis.commands.search.indexDefinition import IndexDefinition, IndexType
+from redis.commands.search.query import Query
+from sentence_transformers import SentenceTransformer
+
+# STEP_END
+
+# STEP_START get_data
+URL = ("https://raw.githubusercontent.com/bsbodden/redis_vss_getting_started"
+       "/main/data/bikes.json"
+       )
+response = requests.get(URL, timeout=10)
```

Comment:
Maybe this data should be included here instead of being pulled from another repo that can change at any moment. That'd drop the `requests` dependency.

Reply:
@akx Thanks for the review - very much appreciated! I did wonder myself if it would be better to add the JSON to the code. However, I think the [tutorial](https://redis.io/docs/latest/develop/get-started/vector-database/#1-fetch-the-demo-data) gets the data from a URL to illustrate the kind of task the user might want to do in real life. Also, the file on the Github repo was deliberately created for this tutorial, so it shouldn't change unless we want it to. I'd say we should leave it like this for now but I'll find out if we've got any feedback about this page and see if the users have any problems with the HTTP request.